modules/fts/pages/fts-searching-with-sdk.adoc

```diff
@@ -13,54 +13,57 @@ Couchbase provides several SDKs to allow applications to access a Couchbase clus
 |===
 | SDK | Details | Link
 
-| Java SDK
-| The Java SDK forms the cornerstone of our JVM clients.
-It allows Java applications to access a Couchbase Server cluster.
-The Java SDK offers traditional synchronous APIs and scalable asynchronous APIs to maximize performance
-|xref:3.1@java-sdk:hello-world:overview.adoc[Java SDK 3.2]
-
-| Scala SDK
-| Our new Scala SDK allows Scala applications to access a Couchbase Server cluster.
-It offers synchronous, asynchronous, and reactive APIs for flexibility and maximum performance.
-|xref:1.1@scala-sdk:hello-world:overview.adoc[Scala SDK 1.2]
-
 | .NET SDK
 | The .NET SDK enables you to interact with a Couchbase Server cluster from the .NET Framework using any Common Language Runtime (CLR) language, including C#, F#, and VB.NET. 
 It offers both a traditional synchronous API and an asynchronous API based on the Task-based Asynchronous Pattern (TAP).
 []
-|xref:3.1@dotnet-sdk:hello-world:overview.adoc[.NET SDK 3.2]
+|xref:3.3@dotnet-sdk:howtosfull-text-search.adoc[.NET SDK 3.3]
 
 |C SDK
 |The Couchbase C SDK (`libcouchbase`) enables C and C++ programs to access a Couchbase Server cluster.
 The C SDK is also commonly used as a core dependency of SDKs written in other languages to provide a common implementation and high performance.
 Libcouchbase also contains the `cbc` suite of command line tools.
-|xref:3.0@c-sdk:hello-world:start-using-sdk.adoc[C SDK 3.2]
+|xref:3.3@c-sdk:howtosfull-text-search.adoc[C SDK 3.3]
+
+|Go SDK
+|The Couchbase Go SDK allows you to connect to a Couchbase Server cluster from Go.
+The Go SDK is a native Go library and uses the high-performance gocbcore to handle communicating to the cluster over Couchbase's binary protocols.
+|xref:2.5@go-sdk:howtosfull-text-search.adoc[Go SDK 2.5]
+
+| Java SDK
+| The Java SDK forms the cornerstone of our JVM clients.
+It allows Java applications to access a Couchbase Server cluster.
+The Java SDK offers traditional synchronous APIs and scalable asynchronous APIs to maximize performance
```

Comment:
Full stop after "performance"? All the other Detail entries have a full stop.

Reply:
:+1: